pkg/apis/troubleshoot/v1beta2/collector_shared.go

```diff
@@ -123,6 +123,16 @@ type Database struct {
 	URI           string `json:"uri" yaml:"uri"`
 }
 
+type RRD struct {
```

Comment:
How about calling this collector something more generic.  It doesn't really care about the format of the files collected.

Reply:
I think there's discoverability in "collectd" vs `rrd`.  Unless there's another common tool that overlaps with troubleshoot using rrd?

Reply:
Makes sense.